### PR TITLE
feat: support entrypoint/command in harness and feature

### PR DESCRIPTION
**Pull request by stormqueen1990**
Add support to specifying a custom entrypoint and command in both harness and feature.

### CODE REVIEW

File: internal/harnesses/container/container.go [hunk 1]
```diff
@@ -87,18 +90,27 @@ func New(_ context.Context, name string, cfg Config) (types.Harness, error) {
 		})
 	}
 
+	if cfg.Command == nil || len(cfg.Command) == 0 {
```
Comment (joshrwolf) [inline internal/harnesses/container/container.go:93]:
```suggestion
	if len(cfg.Command) == 0 {
```

this should be enough, len(nil) == 0

File: internal/containers/provider/provider.go [hunk 2]
```diff
@@ -12,10 +12,14 @@ import (
 )
 
 type ExecConfig struct {
-	// The command to be executed in the provider
+	// The "entrypoint" for the exec command. This is not the docker run entrypoint as that command cannot be overridden
+	// at exec time, but rather a command to be prepended to the Command field before running the test.
+	Entrypoint []string
```
Comment (joshrwolf) [inline internal/containers/provider/provider.go:17]: I might be missing this, but can you add a test for this leg?

Comment (stormqueen1990) [reply internal/containers/provider/provider.go]: I removed this as it wasn't adding a lot of value.

**Comment by stormqueen1990** [comment]
I think we don't need this anymore.

**Pull request closed without merging**